bpmsuite/bpmsuite70-businesscentral-monitoring-with-smartrouter.json

```diff
@@ -49,173 +63,109 @@
             "required": false
         },
         {
-            "displayName": "Busines Central Monitoring Custom https Route Hostname",
-            "description": "Custom hostname for https service route.  Leave blank for default hostname, e.g.: secure-<application-name>-buscentrmon-<project>.<default-domain-suffix>",
-            "name": "BUSINESS_CENTRAL_MONITORING_HOSTNAME_HTTPS",
-            "value": "",
+            "displayName": "Smart Router listening port",
+            "description": "Port in which the smart router server listens (router property org.kie.server.router.port)",
+            "name": "KIE_ROUTER_PORT",
+            "value": "9000",
             "required": false
         },
         {
-            "displayName": "Database JNDI Name",
-            "description": "Database JNDI name used by application to resolve the datasource, e.g. java:/jboss/datasources/ExampleDS",
-            "name": "DB_JNDI",
-            "value": "java:jboss/datasources/ExampleDS",
+            "displayName": "Smart Router external URL",
+            "description": "Public URL where the router can be found. Format http://<host>:<port> (router property org.kie.server.router.url.external)",
+            "name": "KIE_ROUTER_URL_EXTERNAL",
+            "example": "http://my-app-smartrouter-projectName.osurl:9000",
             "required": false
         },
         {
-            "displayName": "Database Name",
-            "description": "Database name",
-            "name": "DB_DATABASE",
-            "value": "root",
-            "required": true
-        },
-        {
-            "displayName": "Queues",
-            "description": "Queue names",
-            "name": "HORNETQ_QUEUES",
-            "value": "",
+            "displayName": "Smart Router ID",
+            "description": "Router ID used when connecting to the controller (router property org.kie.server.router.id)",
+            "name": "KIE_ROUTER_ID",
+            "value": "kie-server-router",
             "required": false
         },
         {
-            "displayName": "Topics",
-            "description": "Topic names",
-            "name": "HORNETQ_TOPICS",
-            "value": "",
+            "displayName": "Smart Router name",
+            "description": "Router name used when connecting to the controller (router property org.kie.server.router.name)",
+            "name": "KIE_ROUTER_NAME",
+            "value": "KIE Server Router",
             "required": false
         },
         {
-            "displayName": "Server Keystore Secret Name",
-            "description": "The name of the secret containing the keystore file",
-            "name": "HTTPS_SECRET",
-            "value": "bpmsuite-app-secret",
+            "displayName": "Smart Router controller URL",
+            "description": "URL to access the controller (standalone or embedded in the business central) (router property org.kie.server.controller)",
+            "name": "KIE_ROUTER_CONTROLLER_URL",
```

Comment:
KIE_ROUTER_CONTROLLER_URL ->  KIE_SERVER_CONTROLLER_PROTOCOL , KIE_SERVER_CONTROLLER_SERVICE , KIE_SERVER_CONTROLLER_HOST and KIE_SERVER_CONTROLLER_PORT

Reply:
Correct. You can see how these different variables are used here: https://github.com/jboss-openshift/cct_module/blob/master/os-bpmsuite-smartrouter/added/launch/bpmsuite-smartrouter.sh#L93